e2e/cypress/integration/multi_team_and_dm/gm_add_user_spec.js

```diff
@@ -70,6 +90,54 @@ describe('Multi-user group messages', () => {
             should('be.visible').
             and('contain.text', 'You can add 6 more people');
     });
+
+    it('MM-T468 Group Messaging: Add member to existing GM', () => {
+        cy.apiLogin(testUser);
+
+        // # add some messages to the channel
+        cy.visit(`/${testTeam.name}/channels/${groupChannel.name}`);
+        cy.postMessage('some');
+        cy.postMessage('historical');
+        cy.postMessage('messages');
+
+        // # Open add member modal
+        cy.get('#channelHeaderDropdownButton button').click();
+        cy.get('#channelAddMembers button').click();
+
+        // * verify message says: "This will start a new conversation. If you're adding a lot of people, consider creating a private channel instead."
+        cy.get('#moreDmModal').should('be.visible');
+        const warnMessage = 'This will start a new conversation. If you\'re adding a lot of people, consider creating a private channel instead.';
+        cy.contains(warnMessage).should('be.visible');
+
+        // * verify users are listed in the GM box
+        userList.forEach((user) => {
+            cy.get('.react-select__multi-value div').should('contain', user.username);
+        });
+
+        // # Type a search term and select an autocomplete option.
+        cy.get('#selectItems input').click().type('beatrice');
+        cy.get('.loading-screen').should('not.be.visible');
+        cy.contains('#multiSelectList .clickable', 'beatrice').should('be.visible'); // .click(); runs into dettached dom element
+        cy.get('#selectItems input').type('{enter}');
+
+        // # Click Go
+        cy.get('button#saveItems').click({force: true});
+
+        // * modal closes
+        cy.get('#moreDmModal').should('not.be.visible');
+
+        // * original messages does not exist
+        cy.contains('.post-message__text', 'historical').should('not.be.visible');
```

Comment:
One of the expected test steps is that it. should show. the message `This is the start of your group message history...".` after creating the group. Can you add it?